tests/api_tests.js

```diff
@@ -322,8 +323,13 @@ function runNodeEnvTests(nodeEnvData, t) {
                                          t.ok(parseInt(nodeEnvData['max.old.space.size']) > 0,
                                               "max.old.space.size is positive");
 
-                                              var nodeVersion = Number(process.version.match(/^v(\d+\.\d+)/)[1]);
-                                              if(nodeVersion >= 6.5) { // issue 283
+                                              // converts e.g. 'v8.0.0' to [8,0.0]
```

Comment:
Also, I think I've read a half dozen buggy attempts to compare versions by now, yesterday was https://github.com/ibmdb/node-ibm_db/pull/228#discussion_r102289716. I've looked at yours, and I think it is correct, but it really would be better if people just use `semver` unless there is a really compelling reason not to (like the fact that package dependencies cannot be relied on to be installed before an install script is run in npm@2, but that doesn't apply here).